client/foundries_pki.go

```diff
@@ -19,11 +19,6 @@ type CaCerts struct {
 	TlsCsr  string `json:"tls-csr"`
 
 	ChangeMeta ChangeMeta `json:"change-meta"`
-
-	CreateCaScript       *string `json:"create_ca"`
```

Comment:
Is this require corresponding change in the PKI creation handler at ota-lite so it doesn't return those scripts?

Reply:
Not yet.

The `ota-lite` needs to remain unmodified to support old clients.
Once the new version is released and proliferate to enough users (allowing enough time for that), the API code will be cleaned up.